prisma/schema.prisma

```diff
@@ -0,0 +1,20 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
```

Comment:
Tem como o provider ser um valor do env também?

Reply:
Aparentemente não, aparece a mensagem: A datasource must not use the env() function in the provider argument.

Reply:
Podemos investigar numa próxima feature